docs/snippets/common/storybook-test-runner-a11y-disable.js.mdx

```diff
@@ -9,7 +9,7 @@ const { getStoryContext } = require('@storybook/test-runner');
  * See https://storybook.js.org/docs/7.0/react/writing-tests/test-runner#test-hook-api-experimental
  * to learn more about the test-runner hooks API.
  */
-module.exports = {
+export default {
```

Comment:
here too

Reply:
Thanks for catching this! the test runner should not be included in this PR